scripts/sca_integration/sca_export.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
The file is a generated file and will be removed from the commit sooner or later.
It is here to illustrate the output format.

Question:
Why are CMAKE_CXX_COMPILER and all the other variables empty except for the CMAKE_C_COMPILER?
Did I include the sca_integration.cmake too early?
Any hint for me?